### ISSUE TITLE
Emote credits stuck at 49, cannot go up

### ISSUE THREAD

**Issue by meaninglesshours** [cmooref17/Lethal-Company-TooManyEmotes#131]
I've messed around with all the configs- and I can't seem to figure out how to get emote credits during a game and without party pooper.

**Comment by cmooref17** [comment]
It should go up any time you gain normal game credits. So when you sell items, you should gain a percentage of the credits you get as emote credits as well. Can you confirm that this does not happen?

**Comment by meaninglesshours** [comment]
Yes. Once we sold items, our credits were still stuck at 49 credits. We do not gain any emote credits on sell.

**Comment by cmooref17** [comment]
Does this still happen? And does it happen every time you sell stuff?
When it does happen, does it happen to the host?

I had globel unlocks enabled, so I started off with 49 credits. I ran through a game, grabbed some items, and sold them at the company, and I went up to 69 emotes credits.

I'm not sure why it wouldn't work for you. I'm wondering if another mod you had gives you emote credits in a different way.

**Comment by Makarion77** [comment]
Hello, 
I have the same issue, the emotes credits are blocked at 36. Maybe it's because of the mod "SellFromTerminal v1.1.2 by stormytuna" that I use. I'll try to desactive it next time. I'll let you know.
Thanks for all the emotes btw that's reaaaallllyy cool !
